orm/table_create_test.go

```diff
@@ -84,12 +84,19 @@ type CreateTableWithHashPartition struct {
 	String    string
 }
 
+type CreateTableWithMultipleNamedUniques struct {
+	ID               int    `sql:",pk"`
+	AccountID        int    `sql:",unique:'per_account,per_store'"`
```

Comment:
This looks good enough, but I am thinking that maybe it should be `sql:",unique:per_account,unique:per_store'"` and `sqlTag.Options` should be `map[string][]string` - i.e. support multiple values. Do you have any objections?

Reply:
Nevermind. I guess I am overthinking it. Please just use unquote and leave the rest as is.